### DISCUSSION TITLE
Error LNK2019: FEXIST comoiling Anura3D

### DISCUSSION THREAD

**Discussion by alejosfriso** [Anura3D/Anura3D_OpenSource#277] [Q&A]
Dear community
Attempting to compile Anura3D, I got the following error:

Error LNK2019: unresolved external symbol FEXIST referenced in function FEXIST.t690p.t3v

I am using VSC 17.12.1 and the following four intel packages:
- intel-onemkl-2025.0.1.15_offline
- intel-ipp-2022.0.0.796_offline
- intel-oneapi-base-toolkit-2025.0.0.882_offline
- intel-oneapi-hpc-toolkit-2025.0.0.822_offline

May I get some advice on how to fix this?

Alejo Sfriso

**Comment by ghost** [comment]
I am facing the same problem.😭
